### ISSUE TITLE
Consider an extension that allows custom block level tags to be configured in mkdocs.yml

### ISSUE THREAD

**Issue by perezd** [facelessuser/pymdown-extensions#524]
Given this functionality: https://github.com/Python-Markdown/markdown/issues/575
It'd be really nice to be able to build themes which can use custom block level tags. For instance, I'm trying to use this: https://github.com/rsms/raster

It defines custom tags `grid` and `c` and I'd like to declare them as block level so I can apply `markdown=1` to them. 

What do you think about this proposal?

**Comment by facelessuser** [comment]
I think the suggestion has merit, but this project supports enough custom extensions already. This project is mainly extensions I wrote for myself, but freely share. It was never meant to become a warehouse for all extensions. While I do listen to the community to make existing things more flexible and improve things, I don't often add new extensions unless I feel it is something I have a strong need for. 

These are extensions that I frequently use. I occasionally may add new extensions, but that only really occurs when I feel I have a real use for it in my document workflow. Historically when I've added a new extension that has been suggested, it is because I'd been thinking about something similar already.

It is strongly encouraged that people create their own 3rd party extensions for things that pymdown-extensions doesn't already cover. Unfortunately, I just don't have time to support everyone's custom ideas, even if they are good ideas as I'm just one guy, and every thing I accept into the project, I have to support :slightly_smiling_face:.
